gcloud-java-core/src/test/java/com/google/gcloud/SerializationTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud;
+
+import com.google.common.collect.ImmutableList;
+
+import java.io.Serializable;
+
+public class SerializationTest extends BaseSerializationTest {
+
+  private static final PageImpl<String> PAGE =
+      new PageImpl<>(null, "cursor", ImmutableList.of("string1", "string2"));
+  private static final RetryParams RETRY_PARAMS = RetryParams.defaultInstance();
```

Comment:
And we could also add in the AuthCredentials' restorable states.  (we can create an issue and do this later if you prefer not to add it in this PR).